app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties

```diff
@@ -239,6 +239,7 @@ prefs.cssanalyzer.columns.order = CSS Analyzer Columns order :
 prefs.cssanalyzer.columns.defaults.first = "Defaults" Column First
 prefs.cssanalyzer.columns.defaults.last = "Defaults" Column Last
 prefs.recent.items = Recent items
+prefs.animate.accordion = Animate Accordion
```

Comment:
I'd say: `Enable accordion animations`?

Reply:
It's a checkbox. Isn't "enable" implied? :)